src/implementations/vanilla/components/text-area/_text-area.scss

```diff
@@ -0,0 +1,110 @@
+/**
+ * Textarea component
+ * @define text-area
+ */
+
+@use "sass:map";
+@use '@ecl/theme-dev/theme';
+@use '@ecl/vanilla-layout-grid/mixins/breakpoints';
+
+$_border-radius: null !default;
+$_border-color: null !default;
+$_border-color-hover: null !default;
+$_border-color-focus: null !default;
+$_border-color-invalid: null !default;
+$_border-color-invalid-hover: null !default;
+$_border-width: 1px;
+$_border-width-focus: null !default;
+$_border-width-focus: 3px;
+$_color-input: null !default;
+$_color-placeholder: null !default;
+$_shadow: null !default;
+$_shadow-hover: null !default;
+$_padding-vertical: map.get(theme.$spacing, 's');
+$_padding-horizontal: null !default;
+
+.ecl-text-area {
+  appearance: none;
+  background-color: map.get(theme.$color, 'white-100');
+  border: $_border-width solid $_border-color;
+  border-radius: $_border-radius;
+  box-shadow: $_shadow;
+  box-sizing: border-box;
+  color: $_color-input;
+  display: block;
+  font: map.get(theme.$font, 'm');
+  margin: 0;
+  max-width: 100%;
+  padding: calc(#{$_padding-vertical} - #{$_border-width})
```

Comment:
Considering the differences in the border sizes in the different states i wonder a bit about the reason why we use border-box here, it forces us in making these crazy calculations and it actually alters the padding applied which is never corresponding to the one in the specs, but ok...

Reply:
yes I think this is to prevent all the elements below in the page from moving if we focus or different states.